packages/tags/README.md

```diff
@@ -0,0 +1,26 @@
+# @zendeskgarden/react-tags
```

Comment:
Missing badges here. See https://github.com/zendeskgarden/react-components/blob/master/packages/.template/README.md for details.

Reply:
Updated along with other changes.